### ISSUE TITLE
PathNet: Evolution Channels Gradient Descent in Super Neural Networks

### ISSUE THREAD

**Issue by icoxfog417** [arXivTimes/arXivTimes#191]
## 一言でいうと

同構造のネットワーク同士で、互いの学習結果を壊さず相手の学習結果を取り込むことを目指した研究。学習と同時にモジュール(畳込層など)を結ぶパスを遺伝的アルゴリズムで進化させていき、学習が完了したら重みを固定し次のタスクに入る。これで効果的な転移が可能なことを確認(画像&強化学習)。

### 論文リンク

https://arxiv.org/abs/1701.08734

### 著者/所属機関

Chrisantha Fernando, Dylan Banarse, Charles Blundell, Yori Zwols, David Ha, Andrei A. Rusu, Alexander Pritzel, Daan Wierstra
Google DeepMind
Google Brain

**Comment by icoxfog417** [comment]
分かりやすい解説記事

[DeepMind’s PathNet: A Modular Deep Learning Architecture for AGI](https://medium.com/intuitionmachine/pathnet-a-modular-deep-learning-architecture-for-agi-5302fcf53273#.xu1kczdou)
